src/main/plots/save_statistics.py

```diff
@@ -0,0 +1,153 @@
+                 labels=labels)
+    fig.update_traces(textposition='inside', textinfo='percent')
+    __save_plot(fig, path, stat_const.PLOT_TYPES.PIE.value, plot_name, format, auto_open)
+
+
+def __save_plot(fig, path: str, plot_type: stat_const.PLOT_TYPES,
+                plot_name='result_plot', format=stat_const.OUTPUT_FORMAT.HTML.value, auto_open=False):
+    file_name = __create_directory_for_plots(path, plot_type, change_extension_to(plot_name, format))
+    plotly.offline.plot(fig, filename=file_name, auto_open=auto_open)
+
+
+def __plot_bar_chart(statistics_df: pd.DataFrame, column: stat_const.STATISTICS_COLUMN,
+                     title: str, labels: dict,
+                     path: str, plot_name='result_plot', format=stat_const.OUTPUT_FORMAT.HTML.value,
+                     auto_open=False, x_category_order=stat_const.PLOTTY_CATEGORY_ORDER.TOTAL_ASCENDING.value):
+    # x_category_order='total ascending' means: in order of increasing values in Y
+    # x_category_order='category ascending' means: in order of increasing values in X
+    fig = px.bar(statistics_df, x=column, y=stat_const.STATISTICS_COLUMN.FREQ.value, title=title, labels=labels,
+                 hover_data=[column, stat_const.STATISTICS_COLUMN.FREQ.value])
+    fig.update_layout(
+        yaxis=dict(
+            title_text=stat_const.STATISTICS_SHOWING_KEY.FREQ.value
+        ),
+        xaxis=dict(
+            title_text=__get_showing_key(str(column)),
+            # We use type = 'category' because we want to display all values (numbers and strings)
+            type='category',
+            categoryorder=x_category_order
+        ),
+        plot_bgcolor=stat_const.STATISTICS_COLORS.BAR_CHART_BG.value
+    )
+    fig.update_yaxes(automargin=True)
+    __save_plot(fig, path, stat_const.PLOT_TYPES.BAR.value, plot_name, format, auto_open)
+
+
+def __get_labels_for_plots(column: stat_const.STATISTICS_COLUMN):
+    return {
+        stat_const.STATISTICS_COLUMN.FREQ.value: stat_const.STATISTICS_SHOWING_KEY.FREQ.value,
+        column: __get_showing_key(str(column))
+    }
+
+
+def __get_title_for_plots(column: stat_const.STATISTICS_COLUMN):
+    return __get_showing_key(str(column)) + ' distribution'
+
+
+def __get_default_value_for_plots(column: stat_const.STATISTICS_COLUMN):
+    if column == stat_const.STATISTICS_COLUMN.AGE.value:
+        return consts.DEFAULT_VALUES.AGE.value
+    if column == stat_const.STATISTICS_COLUMN.EXPERIENCE.value:
+        return consts.DEFAULT_VALUES.EXPERIENCE.value
+    return None
+
+
+def __get_statistics_info_for_plots(column: stat_const.STATISTICS_COLUMN):
+    return {
+        stat_const.STATISTICS_INFO_FOR_PLOTS.LABELS.value: __get_labels_for_plots(column),
+        stat_const.STATISTICS_INFO_FOR_PLOTS.TITLE.value: __get_title_for_plots(column)
+    }
+
+
+def plot_statistics(file: str, column: stat_const.STATISTICS_COLUMN, plot_type: stat_const.PLOT_TYPES,
```

Comment:
It fails when we pass STATICTIC_COLUMN.FREQ.value in 'column' argument. Maybe it would be better to create a separate variable for FREQ and remove it from STATICTIC_COLUMN?

Reply:
I have changed STATICTIC_COLUMN to STATISTIC_KEY and add a new value STATISTIC_FREQ for FREQ